docs/network/fees.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Fee structure
+---
+
+Etherlink transactions include two fees:
+
+- The _gas fee_ changes depending on Etherlink throughput over time.
+As the demand for Etherlink transactions increases, the gas fee rises.
+- The _storage fee_ is proportional to the amount of data that Etherlink stores on Tezos layer 1.
+Transactions that use more data pay a higher fee.
+
+Unlike some other chains, Etherlink does not use a voluntary gas fee, also known as a tip, to encourage block producers to include transactions more quickly.
+Because the Etherlink sequencer orders transactions in first-come-first-serve order, there is no need to offer higher fees for faster inclusion.
+
+## Gas fee
+
+The gas fee changes based on the transaction throughput over time.
+
+Etherlink measures throughput in terms of _ticks_, which are processing operations.
+As demand increases, Etherlink increases the number of ticks per second to process more transactions in the same amount of time.
```

Comment:
Is this technically accurate? Etherlink is fundamentally not really doing anything different here - it's more that it's requested to increase the number of execution steps/second that it processes. But the rollup node is not guaranteed to actually be able to do that. But the rollup applies back-pressure by increases the gas price.